documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -98,6 +98,12 @@ GO
 ----
 If the result is empty then please make sure that the user has privileges to access both the capture instance and _CDC_ tables.
 
+=== Note on Net Changes support
+While enabling CDC for a table, SQL Server creates a CDC table to store the changes and a function to enumerate all changes. If the parameter @supports_net_changes is set to 1, SQL Server will add to the change table an index and create a second function to enumerate net changes, which means that only the final result of all the captured changes is returned, rather than all the intermediate changes that the first function returns.
```

Comment:
>  only the final result of all the captured changes is returned

What is the scope of this? A transaction, some time period? It's a bit unclear to me what "net changes" exactly means. Could you clarify?

Reply:
Hello, the scope of this is querying the CDC table, in the particular case of a primary been updated multiple time between two LSN. Microsoft gives two functions to query it: 
- You can either use the "all changes" function, which will enumerate all the changes that happenned for that PK.
- Or the "net changes" function, which, for each PK, will return only one row, the one that occured the most recently. Supporting this "net change function has a cost".
- You can of course query it manually.

I my self needed to know the way Debezium queryies SQL server to configure property the CDC, I figured out that it only uses the all change function (ServerConnection.java), and given that enabling "net change support" has a cost", I wanted to write a paragraphe on it to save the time for the ones that might have the same question that I had.

Should I update the paragraphe to make the explanation of net changes clearer?

Reply:
Or may be this paragraphe can be deleted, the one below is enough, and people can read direct the Microsoft doc more more info on how Sql Server manages CDC.

Reply:
> Should I update the paragraphe to make the explanation of net changes clearer?

Yes, that'd be much appreciated!

Reply:
Ok, I'll do that this evening.